src/components/Text/Link.js

```diff
@@ -1,38 +1,112 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-import { colors } from "../../theme";
+import Base from "./Base.styles";
 import StyledText from "./StyledText";
+import colors from "../../theme/colors";
 import getRelByTarget from "../../utils/link";
+import { getFontColor } from "../../utils/typography";
 
 export const LinkTitle = StyledText.withComponent(`a`).extend`
   color: ${({ color }) => color || colors.azure.base};
   text-decoration: none;
 `;
 
-const Link = ({ href, children, ...props }) => {
+const LinkBase = Base.withComponent("a").extend`
```

Comment:
Do we want to return a  `button` when an `onClick` handler is passed? Or a `span` when no `href` nor an `onClick` handler is passed?